contracts/Erc20Vault.sol

```diff
@@ -233,7 +239,9 @@ contract Erc20Vault is
         internal
         returns (bool)
     {
-        weth.withdraw(_tokenAmount);
+        weth.approve(wEthUnwrapperAddress, _tokenAmount);
```

Comment:
I think it's worth adding a note in here as to _why_ we're using an unwrapping contract, since it might not be clear in a year's time. Just mention that the istanbul hardfork increased some opcode's gas-costs which broke this, hence needing an unwrapper.

Reply:
Right, on it